src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewStatisticsClassificationAttributesOutput.java

```diff
@@ -47,21 +69,259 @@
+                    model.getClassificationModel().setCurrentConfiguration(configCurrent);
+                    updateTable(configCurrent);
+                    controller.update(new ModelEvent(ViewStatisticsClassificationAttributesOutput.this,
+                                                     ModelPart.STATISTICAL_CLASSIFIER,
+                                                     configCurrent));
+                }
+            }
+        });
+
+        // Table for displaying classifier parameters/values
+        this.table = SWTUtil.createTableDynamic(root, SWT.BORDER | SWT.V_SCROLL | SWT.H_SCROLL | SWT.FULL_SELECTION);
+        this.table.setLayoutData(SWTUtil.createFillGridData(2));
+        this.table.setLinesVisible(true);
+        this.table.setHeaderVisible(true);
+        this.table.addSelectionListener(new SelectionAdapter() {
+            @Override
+            public void widgetSelected(SelectionEvent arg0) {
+                // TODO prompt edit dialog
+            }
+        });
+        DynamicTableColumn c = new DynamicTableColumn(this.table, SWT.LEFT);
+        c.setWidth("50%", "100px");
+        c.setText(Resources.getMessage("ViewClassificationAttributes.5")); //$NON-NLS-1$
+        c = new DynamicTableColumn(this.table, SWT.LEFT);
+        c.setWidth("50%", "100px");
+        c.setText(Resources.getMessage("ViewClassificationAttributes.6")); //$NON-NLS-1$
+        for (final TableColumn col : this.table.getColumns()) {
+            col.pack();
+        }
+        SWTUtil.createGenericTooltip(this.table);
+    }
+    
+    /**
+     * Creates table items for each parameter of a logistic regression
+     * configuration.
+     * 
+     * @param config
+     */
+    private void createItemsForLogisticRegression(ARXLogisticRegressionConfiguration config) {
+        TableItem item = new TableItem(this.table, SWT.NONE);
+        item.setText(0, Resources.getMessage("ViewClassificationAttributes.11")); //$NON-NLS-1$
+        item.setText(1, String.valueOf(config.getAlpha()));
+
+        item = new TableItem(this.table, SWT.NONE);
+        item.setText(0, Resources.getMessage("ViewClassificationAttributes.12")); //$NON-NLS-1$
+        item.setText(1, String.valueOf(config.getDecayExponent()));
+
+        item = new TableItem(this.table, SWT.NONE);
+        item.setText(0, Resources.getMessage("ViewClassificationAttributes.13")); //$NON-NLS-1$
+        item.setText(1, String.valueOf(config.isDeterministic()));
+
+        item = new TableItem(this.table, SWT.NONE);
+        item.setText(0, Resources.getMessage("ViewClassificationAttributes.14")); //$NON-NLS-1$
+        item.setText(1, String.valueOf(config.getLambda()));
+
+        item = new TableItem(this.table, SWT.NONE);
+        item.setText(0, Resources.getMessage("ViewClassificationAttributes.15")); //$NON-NLS-1$
+        item.setText(1, String.valueOf(config.getLearningRate()));

```

Comment:
I'm creating a tooltip on the table via SWTUtil.createGenericTooltip(this.table);, therefore I don't need to set the tooltip on each item separately.